File: app/src/test/java/eu/kanade/tachiyomi/CustomRobolectricGradleTestRunner.java
package eu.kanade.tachiyomi;

import org.junit.runners.model.InitializationError;
import org.robolectric.RobolectricGradleTestRunner;
import org.robolectric.annotation.Config;
import org.robolectric.manifest.AndroidManifest;

public class CustomRobolectricGradleTestRunner
        extends RobolectricGradleTestRunner {

    public CustomRobolectricGradleTestRunner(Class<?> klass) throws InitializationError {
        super(klass);
    }

    @Override
    protected AndroidManifest getAppManifest(Config config) {
        AndroidManifest androidManifest = super.getAppManifest(config);
        androidManifest.setPackageName("eu.kanade.tachiyomi");
        return androidManifest;
    }
}

File: app/src/test/java/eu/kanade/tachiyomi/CustomRobolectricGradleTestRunner.kt
package eu.kanade.tachiyomi

import org.robolectric.RobolectricTestRunner
import org.robolectric.annotation.Config
import org.robolectric.manifest.AndroidManifest

class CustomRobolectricGradleTestRunner(klass: Class<*>) : RobolectricTestRunner(klass) {

    override fun getAppManifest(config: Config): AndroidManifest {
        return super.getAppManifest(config).apply { packageName = "eu.kanade.tachiyomi" }
    }
}
